src/main/java/com/esri/samples/localserver/local_server_map_image_layer/LocalServerMapImageLayerSample.java

```diff
@@ -75,7 +75,7 @@ public void start(Stage stage) throws Exception {
       server.addStatusChangedListener(status -> {
         if (status.getNewStatus() == LocalServerStatus.STARTED) {
           // start map image service
-          String mapServiceURL = new File("./samples-data/local_server/RelationshipID.mpk").getAbsolutePath();
+          String mapServiceURL = new File("./samples-data/local_server/USCitiesStates.mpk").getAbsolutePath();

```

Comment:
Master sample is fine. Not for sure if new one is approved for samples. I can ask @gayl0000 and see if she knows.